src/test/java/uk/gov/pay/products/resources/PaymentResourceTest.java

```diff
@@ -17,9 +18,7 @@
 
 import static java.lang.String.format;
 import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
-import static org.hamcrest.Matchers.hasEntry;
-import static org.hamcrest.Matchers.hasKey;
-import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.*;
```

Comment:
grrrrr

Reply:
Whats wrong with that. In fact its intelliJ magic

Reply:
Well, we had an agreement with the team sometime ago that we don't use `import xxx.*`